plugins/faustwp/faustwp.php

```diff
@@ -44,11 +44,11 @@ function faustwp_minimum_php_requirement() {
  *
  * @return bool True if meets minimum requirements, false otherwise.
  */
-function faustwp_meets_php_requirements() {
+function is_php_version_compatible() {
 	return version_compare( phpversion(), faustwp_minimum_php_requirement(), '>=' );
 }
 
-if ( ! faustwp_meets_php_requirements() ) {
```

Comment:
@theodesp Sorry, I meant that we can use internal WP function [faustwp_meets_php_requirements](https://developer.wordpress.org/reference/functions/is_php_version_compatible/).

Reply:
Thanks I will use that instead.